src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -1312,8 +1309,7 @@ TxQ::getMetrics(Config const& config, OpenView const& view,
         -> boost::optional<Metrics>
 {
     auto const allowEscalation =
-        (view.rules().enabled(featureFeeEscalation,
-            config.features));
+        (view.rules().enabled(featureFeeEscalation));
```

Comment:
Some of these changes remove dependencies from `TxQ::getMetrics` and `TxQ::getAccountTxs`. You can cherry pick https://github.com/ximinez/rippled/commit/aecded7487f0159f16cb6d4bf68599c359b73418